pkg/controller/install_external_secrets.go

```diff
@@ -79,6 +79,16 @@ func (r *ExternalSecretsReconciler) reconcileExternalSecretsDeployment(es *opera
 		return err
 	}
 
+	if err := r.createOrApplyCertificates(es, resourceLabels, recon); err != nil {
+		r.log.Error(err, "failed to reconcile certificates resource")
+		return err
+	}
+
+	if err := r.createOrApplySecret(es, resourceLabels, recon); err != nil {
+		r.log.Error(err, "failed to reconcile secret resource")
+		return err
+	}
+
```

Comment:
should be called before deployments are created. Let's call this before creating validatingwebhook in this func.